recipe/gen_patch_json.py

```diff
@@ -561,15 +561,24 @@ def _gen_new_index(repodata, subdir):
                 i = record["depends"].index("pyarrow >=0.13.0,!=0.14.0,<2")
                 record["depends"][i] = "pyarrow >=0.17.1,<2"
 
-        # distributed <2.11.0 does not work with msgpack-python >=1.0
-        # newer versions of distributed require at least msgpack-python >=0.6.0
-        # so we can fix cases where msgpack-python is unbounded
-        # https://github.com/conda-forge/distributed-feedstock/pull/114
         if record_name == 'distributed':
+            # distributed <2.11.0 does not work with msgpack-python >=1.0
+            # newer versions of distributed require at least msgpack-python >=0.6.0
+            # so we can fix cases where msgpack-python is unbounded
+            # https://github.com/conda-forge/distributed-feedstock/pull/114
             if 'msgpack-python' in record['depends']:
                 i = record['depends'].index('msgpack-python')
                 record['depends'][i] = 'msgpack-python <1.0.0'
 
+            # click 8 broke dask + distributed prior to 2021.5.0.
+            # These have been corrected in PRs
+            # https://github.com/conda-forge/gcsfs-feedstock/pull/32
```

Comment:
This is the right link?

Reply:
Nope forgot to fix that before submitting. Thanks! 😄